connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectMetrics.java

```diff
@@ -30,6 +30,7 @@
 import org.apache.kafka.common.utils.AppInfoParser;
 import org.apache.kafka.common.utils.Time;
 import org.apache.kafka.connect.runtime.distributed.DistributedConfig;
+import org.apache.log4j.LogManager;
```

Comment:
this change is unnecessary, right?

Reply:
thanks! I should double check about this